src/main/java/transformString.java

```diff
@@ -0,0 +1,28 @@
+// Transform the line with "/" to common
+public class transformString {
+    public static String transformString(String line) {
+        String transformedLine = "";
+        String[] wordsListOfLine = line.split(" ");
+        boolean firstTimeOccurence = true; // Indicates the occurrence time of "/"
+
+        if (line.contains("/")) {
+            for (int i = 0; i < wordsListOfLine.length; i++) {
+                if (wordsListOfLine[i].startsWith("/")) {
+                    if (firstTimeOccurence) {
+                        wordsListOfLine[i] = "(" + wordsListOfLine[i].substring(1) + ":";
```

Comment:
No coding standard violation! Good job on naming the variables using camelCase :D